omui/src/components/list/list.tsx

```diff
@@ -11,17 +11,202 @@ import {
 import { Flex } from '../flex';
 import { Box } from '../box';
 
+type ItemProps = {
+  label: string;
+};
+
+type ListTypes =
```

Comment:
No way, let's find a more elegant solution here. I'm fairly sure this has to exist as a type within Chakra...

Reply:
Ideally, this should be only `ordered` and `unordered` for the most part. I don't know... find a better solution than this.